src/exercises/exercise-4-age-to-days.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Function that converts ages to days
+ * @param age - The amount of ages
+ * @returns The converted days from ages, by the amount of days there is for one age
```

Comment:
consider changing age to years

maybe something like this: `The age * 365 (days in a year)`